pkgs/misc/arm-trusted-firmware/default.nix

```diff
@@ -33,7 +33,9 @@ let
       owner = "ARM-software";
       repo = "arm-trusted-firmware";
       rev = "v${version}";
-      sha256 = "sha256-WDJMMIWZHNqxxAKeHiZDxtPjfsfQAWsbYv+0o0PiJQs=";
+      sha256 = {
+        "2.7" = "sha256-WDJMMIWZHNqxxAKeHiZDxtPjfsfQAWsbYv+0o0PiJQs=";
+      }.${version};
```

Comment:
```suggestion
        name = "source-arm-trusted-firmware-${version}";
        sha256 = "sha256-WDJMMIWZHNqxxAKeHiZDxtPjfsfQAWsbYv+0o0PiJQs=";
      };
```

Reply:
Okay, I guess that's another way of fixing it.

Reply:
I just hope some future improvement to ca-derivations doesn't unfix it.  That hash is computed over only the file contents, not including `${version}`.